README.md

```diff
@@ -506,6 +506,26 @@ extra conditions contradict the conditions in the rules, the Transfer can never
 The Receiver must post a comment describing the resolution of these extra conditions with validity as a result, before
 he can accept the Transfer.
 
+# Representation of Nomics
+
+## Mutable Rules
+
+**377** *Representation Theory*
+
+A **Nomic** is a game in which the rules may change. 
+
+A **Nomic game state** is a record of a Nomic, interpretable
```

Comment:
This is vague on purpose. Currently it's not even plain text (it's rendered Markdown).